theforeman.org/pipelines/vars/katello/4.4.groovy

```diff
@@ -0,0 +1,15 @@
+def foreman_version = '3.2'
+def katello_version = '4.4'
+def foreman_el_releases = [
+    'el7',
+    'el8'
+]
+def pipelines = [
+    'install': [
+        'centos7',
+        'centos8-stream',
+    ],
+    'upgrade': [
+        'centos7',
```

Comment:
We can also upgrade on Stream, right? We can install 4.2, then move to 4.3 and then 4.4.

Reply:
I left it out because I didn't see `centos8-stream` in the 4.2 install pipeline.  If that doesn't matter then I can add in `centos8-stream`.